tests/python/relax/frontend/test_onnx_frontend.py

```diff
@@ -0,0 +1,1379 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=unused-argument
+"""
+ONNX testcases
+================
+This file is a test script to test Relax ONNX frontend coverage.
+"""
+
+from typing import Optional, Dict
+
+import numpy as np
+import pytest
+
+import tvm
+import tvm.testing
+from tvm import relax
+
+import onnx
+from onnx import helper, TensorProto, ModelProto, ValueInfoProto, mapping
+import onnxruntime
+
+
+def generate_random_inputs(
+    model: ModelProto, inputs: Optional[Dict[str, np.array]] = None
```

Comment:
```suggestion
    model: ModelProto, inputs: Optional[Dict[str, np.ndarray]] = None
```
afaik the name of the type is `ndarray` here and elsewhere in this file